src/connection_wrap.h

```diff
@@ -0,0 +1,36 @@
+#ifndef SRC_CONNECTION_WRAP_H_
+#define SRC_CONNECTION_WRAP_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include "async-wrap.h"
```

Comment:
It is not strictly necessary, but after reading [Self_contained_Headers](https://google.github.io/styleguide/cppguide.html#Self_contained_Headers) and this sentence:    
> All header files should be self-contained. Users and refactoring tools should not have to  adhere to special conditions to include the header. Specifically, a header should have header  guards and include all other headers it needs.

I understood this as I should include this header because of the types used in the constructor.  I've not changed this yet, but I'm happy to do so, just wanted to ask about this for future reference.